src/yb/util/wait_state.cc

```diff
@@ -46,30 +54,13 @@ WaitStateCode WaitStateInfo::get_state() const {
   return code_;
 }
 
-void WaitStateInfo::set_metadata(AUHMetadata meta) {
-  metadata_ = meta;
-}
-
-AUHMetadata WaitStateInfo::metadata() {
-  return metadata_;
-}
-
-WaitStateCode WaitStateInfo::code() {
```

Comment:
In rocksdb, the threads that do flushes and compactions, some of them might not be doing anything, so I used this method to get the wait state code and check if it's WaitStateCode::Unused to filter them out

Reply:
updated to use get_code()